src/components/Header.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Logo from '../assets/testlogo2.png';
+import './Header.css';
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      active: props.active,
+    };
+  }
+
+  render() {
+    const { active } = this.state;
+    return (
+      <nav className="header">
+        <img className="logo" src={Logo} alt=" " />
+        <h1 className="header-title">Hearthstone Finder</h1>
+        <button
+          type="button"
+          className="Rollmenu"
+          aria-label="Open the navbar"
+          onClick={() => {
+            const newRollMenu = !active;
+            this.setState({ active: newRollMenu });
+          }}
+        >
+          <span className="icon-menu" aria-hidden="true">
+            ☰
+          </span>
+        </button>
+        <ul className={active ? 'nav-list-active' : 'nav-list-inactive'}>
+          <li
+            className={
+              active ? 'nav-list-item-active' : 'nav-list-item-inactive'
```

Comment:
Comme ce ternaire est répété, le mieux est de calculer la valeur juste après avoir "destructuré" active depuis `this.state`, donc à la ligne 16, juste avant le `return` :

```javascript
const listItemClassName = active ? 'nav-list-item-active' : 'nav-list-item-inactive';
```

Et après sur tous les `li` vous pouvez mettre `className={listItemClassName}`.